documentation/src/main/asciidoc/topics/rest_api_v2.adoc

```diff
@@ -94,6 +94,25 @@ include::rest_examples/get_v2_configuration.adoc[]
 |Sets the required format to return content. Supported formats are `application/xml` and `application/json`. The default is `application/json`. See link:#rest_accept[Accept] for more information.
 |===
 
+[[rest_v2_cache_detail]]
+=== Retrieving All the details related to a cache
+
+To retrieve the details of a {brandname} cache, invoke a `GET` request:
+
+[source,options="nowrap",subs=attributes+]
+----
+include::rest_examples/get_v2_all_details.adoc[]
```

Comment:
No need to use an include for just a single line file :)

Reply:
this is happening all the time in the docs!

Reply:
I know, this was a refactor from @oraNod , we since agreed it's not needed.